README.md

```diff
@@ -176,6 +207,63 @@ GTO the artifact file is committed to Git.
 
 </details>
 
+### Deprecating a version
+
+Sometimes you need mark a specific artifact version as a deprecated one,
+signaling that it's not ready to be used any more. You could just delete a git
+tag, but if you want to preserve a history of the actions, you may find
+`gto deprecate` useful.
+
+```console
+$ gto deprecate awesome-model --version v0.0.1
+Created git tag 'awesome-model@v0.0.1!' that deprecates a version.
+```
+
+<details summary="Some details and options">
+
+If you want to deprecate the version by deleting the Git tags itself, you could
+use
+
+```console
+$ gto deprecate awesome-model --version v0.0.1 --delete
+Deleted git tag 'awesome-model@v0.0.1' that published a version.
+Deleted git tag 'awesome-model#prod#1' that assigned a stage.
+Deleted git tag 'awesome-model#prod#2!' that unassigned a stage.
+To push the changes upstream, run:
+git push origin awesome-model@v0.0.1 awesome-model#prod#1 awesome-model#prod#2! --delete
+```
+
+</details>
+
+### Deregistering an artifact
+
+Sometimes you need to need to mark the artifact as "deprecated", usually meaning
+it's outdated and will no longer be developed. To do this, you could run:
+
+```console
+$ gto deregister awesome-model
+Created Git tag 'awesome-model@!' that deregisters the artifact.
+```
+
+<details summary="Some details and options">
+
+If you want to deprecate an artifact by deleting git tags, you'll need to delete
+all of them for the artifact. You could do that with
+
+```console
+$ gto deregister awesome-model --delete
+Deleted git tag 'awesome-model@v0.0.1' that published a version.
+Deleted git tag 'awesome-model#prod#1' that assigned a stage.
+Deleted git tag 'awesome-model#prod#2!' that unassigned a stage.
+To push the changes upstream, run:
+git push origin awesome-model@v0.0.1 awesome-model#prod#1 awesome-model#prod#2! --delete
+```
+
+It looks just the same as `$ gto deprecate awesome-model --delete`, but will
```

Comment:
How about the first Git tag created while registering (the one without versions)? That will also be deleted, right?

Reply:
Yes, it will be deleted if it exists. In this example if doesn't exist. Fixing this! Thanks.